cytosnake/guards/path_guards.py

```diff
@@ -9,20 +9,19 @@
     - valid path strings
 """
 
-from pathlib import Path
+import pathlib
```

Comment:
I agree that `CytoSnake` needs to guard paths! (in reference to an earlier comment about `CytoSnake` guarding file types)

Reply:
Ah okay! Thanks!